cmd/joy/release.go

```diff
@@ -227,14 +227,15 @@ func NewReleaseRenderCmd() *cobra.Command {
 			}
 
 			return render.Render(cmd.Context(), render.RenderOpts{
-				Env:          env,
-				Release:      releaseName,
-				DefaultChart: cfg.DefaultChart,
-				CacheDir:     cfg.JoyCache,
-				Catalog:      cat,
-				IO:           io,
-				Helm:         helm.CLI{IO: io},
-				Color:        color,
+				Env:           env,
+				Release:       releaseName,
+				DefaultChart:  cfg.DefaultChart,
+				CacheDir:      cfg.JoyCache,
+				ChartMappings: cfg.ChartMappings,
```

Comment:
Just unsure about the name... That _could_ work, though I would probably prefer something that emphasizes the values more than the chart, like `ValuesMapping` (and singular form for "Mapping"). But, it can be up for debate! ;)

Reply:
The name is hard to pin down. Its an arbitrary patching of values as defined by joy catalog configuration.

I will go with ValueMapping